## `measuresoftgram --help` quebrando

Não tenho certeza se é algum problema no meu ambiente. 
Mas a flag --help tá quebrando

### Quando rodo no meu ambiente

```bash
durval@golliaz ~/measure-softgram/cli (284-create-init-file)$ python -m venv env

durval@golliaz ~/measure-softgram/cli (284-create-init-file)$ source env/bin/activate

(env) durval@golliaz ~/measure-softgram/cli (284-create-init-file)$ pip install .
Processing /home/durval/measure-softgram/cli
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting inquirer==2.8.0
  Using cached inquirer-2.8.0-py2.py3-none-any.whl (31 kB)
Collecting tabulate==0.8.10
  Using cached tabulate-0.8.10-py3-none-any.whl (29 kB)
Collecting validators==0.20.0
  Using cached validators-0.20.0.tar.gz (30 kB)
  Preparing metadata (setup.py) ... done
Collecting requests==2.28.1
  Using cached requests-2.28.1-py3-none-any.whl (62 kB)
Collecting pytz
  Using cached pytz-2022.2.1-py2.py3-none-any.whl (500 kB)
Collecting termcolor==1.1.0
  Using cached termcolor-1.1.0-py3-none-any.whl
Collecting readchar==2.0.1
  Using cached readchar-2.0.1-py3-none-any.whl (6.9 kB)
Collecting blessed==1.19.0
  Using cached blessed-1.19.0-py2.py3-none-any.whl (57 kB)
Collecting python-editor==1.0.4
  Using cached python_editor-1.0.4-py3-none-any.whl (4.9 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2022.9.14-py3-none-any.whl (162 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting charset-normalizer<3,>=2
  Using cached charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting decorator>=3.4.0
  Using cached decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting wcwidth>=0.1.4
  Using cached wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Collecting six>=1.9.0
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Using legacy 'setup.py install' for validators, since package 'wheel' is not installed.
Building wheels for collected packages: measuresoftgram
  Building wheel for measuresoftgram (pyproject.toml) ... done
  Created wheel for measuresoftgram: filename=measuresoftgram-3.1.5-py3-none-any.whl size=58142 sha256=d8258f46d7d78eed76c336fe7d54e81a43e7593809ece2fbc3d79b2a0e60e538
  Stored in directory: /tmp/pip-ephem-wheel-cache-nf8_lxtu/wheels/50/9b/26/5633fbd208163184e7bbbc01ceee1769c41c96c9127872d3b7
Successfully built measuresoftgram
Installing collected packages: wcwidth, termcolor, readchar, pytz, python-editor, urllib3, tabulate, six, idna, decorator, charset-normalizer, certifi, validators, requests, blessed, inquirer, measuresoftgram
  Running setup.py install for validators ... done
Successfully installed blessed-1.19.0 certifi-2022.9.14 charset-normalizer-2.1.1 decorator-5.1.1 idna-3.4 inquirer-2.8.0 measuresoftgram-3.1.5 python-editor-1.0.4 pytz-2022.2.1 readchar-2.0.1 requests-2.28.1 six-1.16.0 tabulate-0.8.10 termcolor-1.1.0 urllib3-1.26.12 validators-0.20.0 wcwidth-0.2.5

(env) durval@golliaz ~/measure-softgram/cli (284-create-init-file)$ measuresoftgram --help
Traceback (most recent call last):
  File "/home/durval/measure-softgram/cli/env/bin/measuresoftgram", line 5, in <module>
    from src.cli.cliRunner import main
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/cliRunner.py", line 8, in <module>
    from src.cli.commands.parse_generate.parse_generate import parse_generate
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/commands/parse_generate/parse_generate.py", line 6, in <module>
    from src.cli.commands.parse_generate.generate_utils import GenerateUtils
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/commands/parse_generate/generate_utils.py", line 1, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
```

### Quando rodo dentro de um container

```bash
root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# python -m venv env
root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# 
root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# source env/bin/activate
(env) root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# 
(env) root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# pip install .
Processing /tmp/2022-1-MeasureSoftGram-CLI
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting tabulate==0.8.10
  Using cached tabulate-0.8.10-py3-none-any.whl (29 kB)
Collecting inquirer==2.8.0
  Using cached inquirer-2.8.0-py2.py3-none-any.whl (31 kB)
Collecting validators==0.20.0
  Using cached validators-0.20.0.tar.gz (30 kB)
  Preparing metadata (setup.py) ... done
Collecting pytz
  Using cached pytz-2022.2.1-py2.py3-none-any.whl (500 kB)
Collecting requests==2.28.1
  Using cached requests-2.28.1-py3-none-any.whl (62 kB)
Collecting termcolor==1.1.0
  Using cached termcolor-1.1.0-py3-none-any.whl
Collecting blessed==1.19.0
  Using cached blessed-1.19.0-py2.py3-none-any.whl (57 kB)
Collecting readchar==2.0.1
  Using cached readchar-2.0.1-py3-none-any.whl (6.9 kB)
Collecting python-editor==1.0.4
  Using cached python_editor-1.0.4-py3-none-any.whl (4.9 kB)
Collecting charset-normalizer<3,>=2
  Using cached charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting idna<4,>=2.5
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2022.9.14-py3-none-any.whl (162 kB)
Collecting decorator>=3.4.0
  Using cached decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting six>=1.9.0
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting wcwidth>=0.1.4
  Using cached wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Using legacy 'setup.py install' for validators, since package 'wheel' is not installed.
Building wheels for collected packages: measuresoftgram
  Building wheel for measuresoftgram (pyproject.toml) ... done
  Created wheel for measuresoftgram: filename=measuresoftgram-3.1.5-py3-none-any.whl size=50699 sha256=11a2c2f30b19e3ac2d21a2928550e44f451aacdb59f9dcba70ecfa9d3efc031c
  Stored in directory: /root/.cache/pip/wheels/20/ba/15/ac1b66e5f8d40f9794409b7a4c60ecec069d6331bc8d95b9f2
Successfully built measuresoftgram
Installing collected packages: wcwidth, termcolor, readchar, pytz, python-editor, urllib3, tabulate, six, idna, decorator, charset-normalizer, certifi, validators, requests, blessed, inquirer, measuresoftgram
  Running setup.py install for validators ... done
Successfully installed blessed-1.19.0 certifi-2022.9.14 charset-normalizer-2.1.1 decorator-5.1.1 idna-3.4 inquirer-2.8.0 measuresoftgram-3.1.5 python-editor-1.0.4 pytz-2022.2.1 readchar-2.0.1 requests-2.28.1 six-1.16.0 tabulate-0.8.10 termcolor-1.1.0 urllib3-1.26.12 validators-0.20.0 wcwidth-0.2.5

(env) root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# measuresoftgram --help
Traceback (most recent call last):
  File "/tmp/2022-1-MeasureSoftGram-CLI/env/bin/measuresoftgram", line 5, in <module>
    from src.cli.cliRunner import main
ModuleNotFoundError: No module named 'src.cli'
(env) root@e038555e9b13:/tmp/2022-1-MeasureSoftGram-CLI# 
```

## `measuresoftgram init` quebrando


```bash
(env) durval@golliaz ~/measure-softgram/cli (284-create-init-file)$ measuresoftgram init
Traceback (most recent call last):
  File "/home/durval/measure-softgram/cli/env/bin/measuresoftgram", line 5, in <module>
    from src.cli.cliRunner import main
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/cliRunner.py", line 8, in <module>
    from src.cli.commands.parse_generate.parse_generate import parse_generate
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/commands/parse_generate/parse_generate.py", line 6, in <module>
    from src.cli.commands.parse_generate.generate_utils import GenerateUtils
  File "/home/durval/measure-softgram/cli/env/lib/python3.10/site-packages/src/cli/commands/parse_generate/generate_utils.py", line 1, in <module>
    import pandas as pd
ModuleNotFoundError: No module named 'pandas'
```

--- a/src/cli/cliRunner.py
+++ b/src/cli/cliRunner.py
@@ -26,7 +26,7 @@ def setup():
 
     parser_init = subparsers.add_parser(
         "init",
-        help="Create a init file `.measuresoftgram.json` with your default organization, product and repositories"
+        help="Create a init file `.measuresoftgram` with your default organization, product and repositories"
     )
 
     parser_init.add_argument(

--- a/src/cli/commands/parse_init/parse_init.py
+++ b/src/cli/commands/parse_init/parse_init.py
@@ -28,7 +28,7 @@ def parse_init(file_path, host_url):
         with open('.measuresoftgram', 'w') as file:
             file.write(json.dumps(init_data, indent=4))
 
-        print(colored("'.measuresoftgram' init file created with success ...", "green"))
+        print(colored("'.measuresoftgram' init file created with success", "green"))
     except MeasureSoftGramCLIException as error:
         print(colored(f"Error: {error}", "red"))
         return 1

--- a/src/cli/commands/parse_init/utils.py
+++ b/src/cli/commands/parse_init/utils.py
@@ -2,6 +2,7 @@ import json
 from os.path import exists
 
 import validators
+from termcolor import colored
 
 from src.cli.exceptions import exceptions
 from src.clients.service_client import ServiceClient
@@ -90,7 +91,9 @@ def create_entity(url, name, entity):
     response = ServiceClient.make_post_request(url, payload)
 
     if response.status_code == 201:
-        return json.loads(response.text)["id"]
+        entity_id = json.loads(response.text)["id"]
+        print(colored(f"\tCreated {entity} with name {name} and id {entity_id} ...", "blue"))
+        return entity_id
     elif response.status_code == 400:
         raise exceptions.MeasureSoftGramCLIException(
             f"An {entity} with the provided name already exists. "

--- a/tests/unit/test_init.py
+++ b/tests/unit/test_init.py
@@ -58,7 +58,7 @@ def test_init_create_file(mocker):
         parse_init("tests/unit/data/init.json", DUMMY_HOST)
 
         assert EXPECTED_INIT_DATA == read_json(".measuresoftgram")
-        assert "'.measuresoftgram' init file created with success ..." in fake_out.getvalue()
+        assert "'.measuresoftgram' init file created with success" in fake_out.getvalue()
         assert os.path.exists(".measuresoftgram")
 
 

